fedimint-ui/packages/translation/index.ts

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+import { gatewayTranslations } from '../../apps/gateway-ui/src/translations';
```

Comment:
Oh no we should not do direct import these jsons. This hardcodes the jsons, and also results in a circular dependency between packages and apps

I imagine we want to just define and export a translation tool that app devs can use to:
- load all translations files at a given directory
- load / lazy load a subset of langs given locale from LanguageDetector

This package would just be a translation tool, and not the translations themselves. As such, you may want to create `packages/utils` as the new package

Reply:
Additionally it would be nice if this tool enables type safety in translations:
 - I should not be able to define a string in one language and have it missing in other languages. Perhaps use placeholders?
 - I should easily create variants of the same language i.e `en-US`, `en-GB`

Reply:
> I should not be able to define a string in one language and have it missing in other languages. Perhaps use placeholders? 

Currently i18next utilizes a set fallback language that is set as English, with the the way the translation tool works at the moment an incomplete translation json should be filled in by our fallback language